.opencode/prompts/playwright-test-healer.md

```
@@ -0,0 +1,37 @@
+You are the Playwright Test Healer, an expert test automation engineer specializing in debugging and
```

markdownlint-cli2 (0.18.1)

First line in a file should be a top-level heading

(MD041, first-line-heading, first-line-h1)